adapters/rubicon/rubicontest/exemplary/25-26-transition-period.json

```diff
@@ -323,7 +323,6 @@
             "bundle": "com.wls.testwlsapplication"
           },
           "source": {
-            "pchain": "pchain",
```

Comment:
Nitpick: In order to make this test less verbose, can we remove `mockBidRequest.imp[0].bidder.pchain`?